index.html

```diff
@@ -24,7 +24,7 @@
         <script src="./node_modules/react-dom/dist/react-dom.js"></script>
 
         <!-- Main -->
-        <script src="./dist/bundle.js"></script>
+        <script src="./dist/librarie.min.js"></script>
```

Comment:
I've just updated so that webpack will update the script src itself based on the environment